source/Mlos.Python/mlos/Optimizers/BayesianOptimizerConfigStore.py

```diff
@@ -35,6 +39,14 @@
                 MultiObjectiveHomogeneousRandomForest.__name__
             ])
     ).join(
+        subgrid=SklearnRandomForestRegressionModelConfig.CONFIG_SPACE,

```

Comment:
Could we rename CONFIG_SPACE -> parameter_space to make it consistent with other models?